Orange/widgets/unsupervised/owmds.py

```diff
@@ -651,7 +509,7 @@ def __next_step(self):
             self.unconditional_commit()
             self.__draw_similar_pairs = True
             self._update_plot()
-            self.plot.autoRange(padding=0.1, items=[self._scatter_item])
+            #.plot.autoRange(padding=0.1, items=[self._scatter_item])
```

Comment:
Why is this commented? If it is no longer needed, remove. If it needs to be replaced with something, replace.